bin/run-docker

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
Yes, it is intended to run inside the container. I'll add a leading underscore for now.